.circleci/config.yml

```diff
@@ -16,11 +16,11 @@
 defaults: &defaults
   docker:
     # specify the version you desire here (avoid latest except for testing)
-    - image: andrewberry/drupal_tests:0.4.0
+    - image: quay.io/deviantintegral/drupal_tests:latest
```

Comment:
:+1:

Reply:
Hah, I need to point this to a tag. Will follow up.